google/ads/google_ads/client.py

```diff
@@ -505,79 +465,75 @@ def _get_fault_message(self, exception):
                 return None
 
     def _log_successful_request(self, method, customer_id, metadata_json,
-                                request_id, request_json,
-                                trailing_metadata_json, response_json):
+                                request_id, request, trailing_metadata_json,
+                                response):
         """Handles logging of a successful request.
 
         Args:
             method: The method of the request.
             customer_id: The customer ID associated with the request.
             metadata_json: A JSON str of initial_metadata.
             request_id: A unique ID for the request provided in the response.
-            request_json: A JSON str of the request message.
+            request: An instance of a request proto message.
             trailing_metadata_json: A JSON str of trailing_metadata.
-            response_json: A JSON str of the the response message.
+            response: A grpc.Call/grpc.Future instance.
```

Comment:
can it be a grpc.Future or just a grpc.Call?

Reply:
The response object inherits from both classes, so it's technically an instance of both.